alcs-frontend/src/app/services/card/card.dto.ts

```diff
@@ -0,0 +1,44 @@
+import { BaseCodeDto } from '../../shared/dto/base.dto';
+import { ApplicationRegionDto, ApplicationStatusDto } from '../application/application-code.dto';
+import { UserDto } from '../user/user.dto';
+
+export interface ReconsiderationTypeDto extends BaseCodeDto {
+  backgroundColor: string;
+  textColor: string;
+}
+
+export interface CardTypeDto extends BaseCodeDto {}
+
+export interface ReconsiderationDto {
+  highPriority: boolean;
+  status: string;
+  uuid: string; // this is a cardUuid for now
+  board: string;
+  assignee?: UserDto;
+  // TODO: provide actual properties once we have reconsideration
+  applicationFileNumber: string;
+  statusDetails: ApplicationStatusDto;
+  regionDetails: ApplicationRegionDto;
+  typeDetails: ReconsiderationTypeDto;
+  card: CardDto;
+}
+
+export interface CardCreateDto {
+  boardCode: string;
+  typeCode: string;
+}
+
+export interface CardPartialDto {
```

Comment:
This is a bit odd, its not a Partial<CardDto> since this actually has more fields. Should this be UpdateCardDto or Partial<CardDto> or maybe it just needs a better name?